docker/Makefile

```diff
@@ -523,6 +523,18 @@ endif
 ifdef ACCOUNT_CLOAK_KEY
 	BLOCKSCOUT_CONTAINER_PARAMS += -e 'ACCOUNT_CLOAK_KEY=$(ACCOUNT_CLOAK_KEY)'
 endif
+ifdef PROVIDER_URL
+	BLOCKSCOUT_CONTAINER_PARAMS += -e 'PROVIDER_URL=$(PROVIDER_URL)'
+endif
+ifdef CONSENSUS_ADDRESS
+	BLOCKSCOUT_CONTAINER_PARAMS += -e 'CONSENSUS_ADDRESS=$(CONSENSUS_ADDRESS)'
+endif
+ifdef BSC_OMNI_BRIDGE_MEDIATOR
+	BLOCKSCOUT_CONTAINER_PARAMS += -e 'BSC_OMNI_BRIDGE_MEDIATOR=$(BSC_OMNI_BRIDGE_MEDIATOR)'
+endif
+ifdef ETH_OMNI_BRIDGE_MEDIATOR
+	BLOCKSCOUT_CONTAINER_PARAMS += -e 'ETH_OMNI_BRIDGE_MEDIATOR=$(ETH_OMNI_BRIDGE_MEDIATOR)'
+endif
```

Comment:
can you add these env variables there `AMB_BRIDGE_MEDIATORS`, `FOREIGN_JSON_RPC`